spec/javascripts/spec/track_submit.spec.js

```diff
@@ -0,0 +1,85 @@
+describe('A form submit tracker', function() {
+  "use strict";
+
+  var root = window,
+    tracker,
+    element;
+
+  beforeEach(function() {
+    tracker = new GOVUKAdmin.Modules.TrackSubmit();
+  });
+
+  describe('with defaults', function() {
+    beforeEach(function() {
+      element = $('\
+        <div \
+          data-track-action="form-submitted"\
+          data-track-label= "edit-link">\
+          <form class="js-track" method="post">\
```

Comment:
The `js-track` class isn't necessary here. The data attributes could be put directly on the form element and the wrapper left out too.